python/python/ert_gui/shell/simulations.py

```diff
@@ -33,9 +33,18 @@ def ensembleExperiment(self, line):
         simulation_runner = EnkfSimulationRunner(self.ert())
 
         now = time.time()
-
         print("Ensemble Experiment started at: %s" % datetime.now().isoformat(sep=" "))
 
+        print("Create the run path!")
```

Comment:
Remove the `print( ... )` statements - only the final `print("Start simulations")`should be retained.